examples/hello-world.php

```diff
@@ -45,15 +45,10 @@
 $server->start(new class implements RequestHandler {
     public function handleRequest(Request $request): Response
     {
-        $forwardedFor = \implode(', ', \array_map(
-            fn (ForwardedFor $ff) => $ff->getAddress(),
-            $request->getAttribute(ForwardedForMiddleware::class),
-        ));
-
         return new Response(
             status: HttpStatus::OK,
             headers: ["content-type" => "text/plain; charset=utf-8"],
-            body: "Hello, World! " . $forwardedFor,
+            body: "Hello, World! " . $request->getAttribute(ForwardedForMiddleware::class)?->getFor(),
```

Comment:
Should we use `ForwardedForMiddleware::class` or `ForwardedFor::class` for the attribute name?

Reply:
`ForwardedFor::class` seems better. Updated.